src/Categories/Category/Construction/Spans.agda

```diff
@@ -0,0 +1,64 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Categories.Category
+
+-- The 1-Category of Spans
+module Categories.Category.Construction.Spans {o ℓ e} (𝒞 : Category o ℓ e) where
```

Comment:
So there is something related at `Categories.Category.Instance.Span`. It's weird because it is really a "span diagram" that is defined there, whereas here is the more usual construction of a span in a category. That's definitely a bit of a mess (but I'm not going to require you to clean that part up as part of this!)

However, your `Span` here is a proper categorical construction / diagram that should exist in `Diagram.Span` and then `Spans` should indeed be defined here.

Reply:
Definitely agree that the naming is confusing! Will restructure things 🙂

Reply:
Thanks. To be clear: the 1-category you define does belong exactly in this file, it's the definition of `Span` and `Span⇒` that doesn't. And while you're at it, may as well give names to the identity and composition.

Also, now that I'm looking even more deeply: why `dom` and `cod` ? I know that makes sense in a lot of contexts where spans are used, but in general? [I'm not immediately wanting to change them, rather to be convinced that these are good names.]

Reply:
My mental model and main motivation for these things is that "Categories are Monads in Span(C)", so those names were chosen accordingly. I do agree that these names are a bit tied to that example, and may not suit other applications. If anyone has better names I'd love to hear them 🙂